api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](*) and [devices](*) (and more?) who share access to resources in Office365 or in your app. All operations in Microsoft Graph on groups require administrator consent. Using Microsoft Graph, you can perform the following operations and more:
+- create groups
+- check if a user is a member of a group
+- list all groups a user is a member of
+- list the members of a group
+- add and remove members from a group
+- add and remove owners of a group.
+
+**NOTE**
+Groups are only supported for enterprise accounts, and not supported for Microsoft personal accounts.
+
+## Office 365 groups
+Office 365 groups are for people who collaborate on a project or a team. Members share access to Office365 resources including:
+- calendar
+- drive
+- notes
+- plans (beta)
+
+Using the API, your app can access these resources to further drive engaging collaboration scenarios. There are 2 types of Office 365 groups:
+
+| Type | Use case | Shared resources |
+|------------|-------|--------|
+| Outlook group | Mail-driven collaboration | Mail conversations and threads, tasks (beta) |
+| Microsoft Team | High velocity chat-driven collaboration | Channels, chat threads |
+| *TODO* Yammer group | How to access Yammer posts through API? |
+
+**NOTE** Today, only Outlook groups can be created through the API. 
+
+**NOTE** Today, you can check if an Office 365 group is an Outlook group or a Microsoft Teams by trying GET /group/{id}/channels. If it works, then it's a Microsoft Team, else it's an Outlook group.
+
+### Example of Outlook group
+
+```http
+
+{
+    "@odata.context": "https://graph.microsoft.com/beta/$metadata#groups/$entity",
+    "id": "4c5ee71b-e6a5-4343-9e2c-4244bc7e0938",
+    "deletedDateTime": null,
+    "classification": "MBI",
+    "createdDateTime": "2016-08-23T14:46:56Z",
+    "description": "This group will be use to track improvements we make to the Graph IA site",
+    "displayName": "Awesome Graph Docs",
+    "groupTypes": [
+        "Unified"
+    ],
+    "mail": "graphia@service.microsoft.com",
+    "mailEnabled": true,
+    "mailNickname": "graphia",
+    "preferredLanguage": null,
+    "proxyAddresses": [
+        "smtp:graphia@microsoft.onmicrosoft.com",
+        "SMTP:graphia@service.microsoft.com"
+    ],
+    "renewedDateTime": "2016-08-23T14:46:56Z",
+    "securityEnabled": false,
+    "theme": null,
+    "visibility": "Public"
+}
+```
+## Security group and mail-enabled security group
+Security groups are for controlling user access to resources. By checking whether or not a user is a member of a security group, your app can make authorization decisions when that user is trying to access some secure resources in your app. Mail-enabled security groups are the same on principle, but with a group mailbox provisioned by default so the group can receive mail.

```

Comment:
same **in** principle
EXCEPT - I don't think you can create mail-enabled SGs through MS Graph - or can you?